text/message_recovery.go

```diff
@@ -35,7 +35,7 @@ func NewRecoveryEmailSent() *Message {
 	return &Message{
 		ID:      InfoSelfServiceRecoveryEmailSent,
 		Type:    Info,
-		Text:    "An email containing a recovery link has been sent to the email address you provided.",
+		Text:    "An email containing a recovery link has been sent to the email address you provided. If you have not received an email, check the spelling of the address.",
```

Comment:
This is an acceptable workaround. I think that it would be cleaner to have a dedicated message when this feature is enabled though, but I'll leave that up to you.

Reply:
Even if we have notifying unknown recipients enabled, and a user types their email address wrong "joas(at)ory.sh" (instead of "jonas(at)ory.sh") for example, they wouldn't notice either, right? 

So this message applies either way, IMO.

Reply:
ha, good point!

Reply:
in that case though, the message should be: "if you did not receive a message you either have a typo or the address is unknown"?

Reply:
I guess it could be, but what would be the benefit for a user here? Either way, if they think they have an account, but this message shows up, and they're sure they have no typo, they need to contact the administrator of the application, who'd then check in the admin portal or API. 

> An email containing a recovery link has been sent to the email address you provided. If you have not received an email, check the spelling of the address and make sure to use the address you registered with.

Gets quite long, but would work. WDYT?
<img width="437" alt="image" src="https://user-images.githubusercontent.com/5540255/217544631-8cdba8f6-2b7a-4cbe-ad1d-be2bc7dacfd2.png">